.github/workflows/github-actions.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           redis-server --version
           redis-server contrib/redis/sentinel.conf --sentinel
           cd pybossa && ln -s themes/default/translations && cd ..
-          cp settings_test.py.tmpl settings_test.py
+          cp settings_test.py.tmpl pybossa/settings_test.py
```

Comment:
I think the path should be `/pybossa/settings_test.py`, but with this change the path will be `/pybossa/pybossa/settings_test.py`, is this expected?

Reply:
nvm I saw you changed `run_tests`

Reply:
right. also, this was not required but `nice to have` all settings at once place.